collect_app/src/main/java/org/odk/collect/android/preferences/AdminKeys.java

```diff
@@ -0,0 +1,79 @@
+package org.odk.collect.android.preferences;
+
+import static org.odk.collect.android.preferences.AdminAndGeneralKeys.ag;
+
+public final class AdminKeys {
+    // key for this preference screen
+    public static String KEY_ADMIN_PW = "admin_pw";
+
+    // keys for each preference
+
+    // main menu
+    public  static String KEY_EDIT_SAVED                = "edit_saved";
```

Comment:
I'll file it as a quick win for one of our new contributors! 😊 Just wanted to make sure I wasn't missing something.